### PR TITLE
42 update generator tests

### PR DESCRIPTION
**Pull request by codeninja**
updated aurora tests at the integration level to ensure functionality.

### CODE REVIEW

**Comment by codeninja** [comment]
I've updated the test for `should throw imaginary Glob Files` to not display it's error in console as well. 

When expecting an error, use ` spyOn(console, 'error'); ` in the test which expects the thrown error to suppress it from console.

**Comment by codeninja** [comment]
Please let  me know when these updates are pushed to NPM! Eagerly awaiting the combined generators!

TYVM.

File: src/tests/schemas/feature-specific/generators/generator-binary-env.prisma [hunk 1]
```diff
@@ -1,16 +1,8 @@
...
   provider        = "prisma-client-js"
   output          = "../src/generated"
   previewFeatures = ["referentialIntegrity"]
   engineType      = "library"
-  binaryTargets   = env("BINARY_TARGETS")
-}
-
-model User {
-  id Int @id
+  binaryTargets   = ["binary-target"]
```
Comment (sabinadams) [inline src/tests/schemas/feature-specific/generators/generator-binary-env.prisma:7]: This should use an `env` var, as we need to check that scenario is covered

**Comment by codeninja** [comment]
@sabinadams Updated.

I think this is as close as I'm going to be able to get to testing the ENV vars... Prisma is executing the `env()` when it parses the schema. All it looks like I can do is check for the env() within the string which is returned from the generator.

I can't, don't know how to, validate that the env var is ACTUALLY interpreted by Prisma.

**Comment by sabinadams** [comment]
That's exactly what I was looking for thanks! 

I'm not interested in testing Prisma's functionality. That's up to them. Just want to make sure Aurora will render out that `env` function.

**Review by sabinadams — APPROVED** [review]

**Comment by codeninja** [comment]
Awesome! Just making sure you knew what I was giving you :)

Thanks,
Dallas
-    -     -     -     -     -     -     -     -     -     -     -      -
Director of Engineering @ Microventures

*Professional leader of highly performant engineering teams*
*512.923.4426 | Github  <https://github.com/codeninja>| Twitter
<https://twitter.com/dallaspool>| Linkedin
<https://linkedin.com/in/dallaspool>*

**Pull request merged**